mod/zaudio/start.php

```diff
@@ -13,6 +13,12 @@
 function zaudio_init() {
 	elgg_extend_view('css/elgg', 'zaudio/css');
 
+	elgg_define_js('elgg/zaudio/AudioPlayer', [
```

Comment:
This probably shouldn't be prefixed with elgg. It's a third party lib. We don't do elgg/jquery for example